Sources/CryptoSwift/HashProtocol.swift

```diff
@@ -16,6 +16,10 @@ internal protocol HashProtocol: class {
 extension HashProtocol {
     
     func prepare(_ len:Int) -> Array<UInt8> {
+        return self.prepare(len, allowance: 8)
+    }
+    
+    func prepare(_ len:Int, allowance: Int) -> Array<UInt8> {
```

Comment:
What I meant was to simply change `return self.prepare(len, allowance: 8)` to `return self.prepare(len, allowance: len / 8)`.

IMHO, `prepare` is okay for the function name as the RFC implementation calls it `pre-processing`.